### ISSUE TITLE
Wacom Intous Pro inpainting problem

### ISSUE THREAD

**Issue by E2GO** [lllyasviel/stable-diffusion-webui-forge#953]
Problem with drawing masks\sketching using Wacom Intuos Pro (PTH-660). 
The mask starts to draw, but almost immediately stops and starts moving whole screen. Check video. 

```
Loading log:
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: f1.0.2v1.10.1-previous-168-gf743fbff
Commit hash: f743fbff83c7db0bf0957ab9718f8d42a47eb35e
Launching Web UI with arguments: --xformers --always-gpu --disable-nan-check --cuda-stream --pin-shared-memory --ckpt-dir 'J:\Checkpoints' --clip-models-path 'J:\cachenew' --theme dark --embeddings-dir 'J:\A1111\stable-diffusion-webui\embeddings' --lora-dir 'J:\A1111\stable-diffusion-webui\models\Lora'
Total VRAM 24564 MB, total RAM 65414 MB
pytorch version: 2.1.2+cu121
WARNING:xformers:A matching Triton is not available, some optimizations will not be enabled.
Error caught was: No module named 'triton'
xformers version: 0.0.23.post1
Set vram state to: HIGH_VRAM
Always pin shared GPU memory
Device: cuda:0 NVIDIA GeForce RTX 4090 : native
Hint: your device supports --cuda-malloc for potential speed improvements.
VAE dtype preferences: [torch.bfloat16, torch.float32] -> torch.bfloat16
CUDA Stream Activated:  True
J:\webui_forge_cu121_torch21\system\python\lib\site-packages\transformers\utils\hub.py:127: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.
  warnings.warn(
Using xformers cross attention
Using xformers attention for VAE
ControlNet preprocessor location: J:\webui_forge_cu121_torch21\webui\models\ControlNetPreprocessor
[-] ADetailer initialized. version: 24.6.0, num models: 10
2024-08-07 14:23:10,289 - ControlNet - INFO - ControlNet UI callback registered.
Loading weights [c2e61f56f9] from J:\webui_forge_cu121_torch21\webui\models\Stable-diffusion\FluentlyXL-Final.safetensors
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
StateDict Keys: {'unet': 1680, 'vae': 248, 'text_encoder': 197, 'text_encoder_2': 518, 'ignore': 0}
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.: 4.3s, import torch: 9.2s, initialize shared: 0.2s, other imports: 3.0s, list SD models: 3.6s, loa
K-Model Created: {'storage_dtype': torch.float8_e4m3fn, 'computation_dtype': torch.float16, 'manual_cast': True}
Model loaded in 14.4s (load weights from disk: 1.7s, forge model load: 12.7s).
```

https://github.com/user-attachments/assets/bd4fe057-6997-4311-b540-ff97cf187d4a

**Comment by Digiearts** [comment]
I also have exactly same problem with my Surface tablet. I have opened the issue around two weeks ago and its status is still under construction

**Comment by E2GO** [comment]
@lllyasviel Just check feature status and it says: Wacom 128 level touch pressure support for Canvas | Normal | 2024 July 15
FYI Still got same issue on device listed above. Latest build.

**Comment by kievhsiao** [comment]
Same problem with Wacom CTL-480. 

```
Python 3.10.10 (tags/v3.10.10:aad5f6a, Feb  7 2023, 17:20:36) [MSC v.1929 64 bit (AMD64)]
Version: f2.0.1v1.10.1-previous-260-gaadc0f04
Commit hash: aadc0f04c48eb19475752a4206420ea2004e2f42
Launching Web UI with arguments: --theme dark
Total VRAM 4096 MB, total RAM 15613 MB
pytorch version: 2.3.1+cu121
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 3050 Laptop GPU : native
```

**Comment by dimdieo** [comment]
Same problem on ipad

**Comment by kievhsiao** [comment]
I found that in the Wacom settings, Pen Settings => Mapping => unchecking "Use Windows Ink" resolves the issue of interruptions while drawing.
![螢幕擷取畫面 2024-08-24 184538](https://github.com/user-attachments/assets/71eaf682-7a73-4677-99fc-ef89780da39d)
My interface is in Chinese, and I translated it into English. It might be different in the UI of other languages.

**Comment by gilguo95** [comment]
> I found that in the Wacom settings, Pen Settings => Mapping => unchecking "Use Windows Ink" resolves the issue of interruptions while drawing. ![螢幕擷取畫面 2024-08-24 184538](https://github.com/user-attachments/assets/71eaf682-7a73-4677-99fc-ef89780da39d) My interface is in Chinese, and I translated it into English. It might be different in the UI of other languages.

problem solved, thx
